pages/index.html

```diff
@@ -49,7 +57,7 @@
         </div>
         <div class="coveo-main-section">
             <div class="coveo-facet-column">
-                <div class="CoveoAttachedResultsFilter"></div>
```

Comment:
Are we planning on having more than 1 `CoveoFilter*` ?

Reply:
maybe, what is your point 📦

Reply:
@erocheleau would it be better to rename it "FilterResults" only or "FieldFilterCheckbox" somethinglike that?